src/app/views/admissions/student-form/student-utils/ProfileForm.jsx

```diff
@@ -168,6 +201,50 @@ export const ProfileForm = ({ initialValues }) => {
               />
               <small>Only cohorts with stage PREWORK or STARTED will be shown here</small>
             </Grid>
+            {availableAsSaas === true && (
+            <>
+            <Divider className="mb-2" />
+              <Grid item md={12} sm={12} xs={12}>
+                <Alert severity='warning'>
+                  <AlertTitle> On adding a new cohort</AlertTitle>
+                  You are selecting a cohort that is available as saas, in order to add him/her to this cohort, you should select the plan that you want to add him to
+                </Alert>
+              </Grid>
+            <Grid item md={2} sm={4} xs={12}>
+              Plan
+            </Grid>
+            <Grid item md={10} sm={8} xs={12}>
+              <AsyncAutocomplete
+                onChange={(newPlan) => {
+                  setSelectedPlans(newPlan);
+                }}
+                width="30%"
+                size="small"
+                label="Select a plan"
+                debounced={false}
+                isOptionEqualToValue={(option, value) => option.id === value.id}  
+                getOptionLabel={(option) => `${option.slug}`}
+                multiple={false}
+                asyncSearch={() => {
+                  const selectedCohortSlug = cohort.length > 0 ? cohort[0].slug : null;
+                  if (selectedCohortSlug) {
+                    return axios.get(`${config.REACT_APP_API_HOST}/v1/payments/plan?cohort=${selectedCohortSlug}`)
```

Comment:
Add this request in the payments object of requests located in the `/services/breathecode.js` file